src/main/java/org/jboss/threads/EnhancedQueueExecutor.java

```diff
@@ -292,6 +292,13 @@ public final class EnhancedQueueExecutor extends AbstractExecutorService impleme
     @SuppressWarnings("unused") // used by field updater
     volatile int peakQueueSize;
 
+    // =======================================================
+    // Size and queue limit tracking
+    // =======================================================
+
+    private final boolean queueLimit;
+
+
```

Comment:
Extra whitespace here; plus I'd call this `queueLimited` (see below for similar suggestions). Grammatically, "queueLimit" sounds like it would be the number representing the queue size.